docs/source/examples/smoothing.ipynb

```diff
@@ -0,0 +1,338 @@
+{
```

Comment:
<p>This should be something I address in another PR for <code>compute_hindcast</code> in the classes. We shouldn't expect a user to use a private function and the current setup is clunky like this. But it should be an easy fix and this is fine for the pre-release master docs.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/bradyrx/climpred/pull/244/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='6'/>